core/include/rectangle_shape.h

```diff
@@ -11,11 +11,11 @@ namespace rinvid
 class RectangleShape : public FixedPolygonShape<4>
 {
   public:
-    RectangleShape(Vector2 vert1, Vector2 vert2, Vector2 vert3, Vector2 vert4);
+    RectangleShape(Vector2 top_left, Vector2 top_right, Vector2 bottom_right, Vector2 bottom_left);
```

Comment:
Does the order of verts actually matter? Couldn't someone just move "top_left" one to the end and still get "same" rectangle?

Reply:
No, it's a recommended order xD
I will create another PR for refactoring of rectangle class.